### PR TITLE
Rebuild packages for missing metadata vol. 8

### PR DESCRIPTION
**Pull request by Chocimier**
All packages was build (when available) for x86_64, x86_64-musl, armv7hf and aarch64-musl and passed check stage on x86_64 and x86_64-musl. Nothing was tested manually whether it still works.

104 more packages remain to rebuild.

### CODE REVIEW

File: srcpkgs/mousetweaks/template [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-revision=3
+revision=4
 build_style=gnu-configure
 hostmakedepends="pkg-config intltool glib-devel $(vopt_if gir gobject-introspection)"
 makedepends="gsettings-desktop-schemas-devel gtk+3-devel libXtst-devel"
 depends="gsettings-desktop-schemas>=3.12 desktop-file-utils"
 short_desc="Mouse accessibility enhancements for the GNOME desktop"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://www.gnome.org"
 license="FDL, GPL-3"
```
Comment (maxice8) [inline srcpkgs/mousetweaks/template]: GPL-3.0-or-later

File: srcpkgs/mrxvt/template [hunk 2]
```diff
@@ -12,5 +11,5 @@ maintainer="Dan Kociela <dkociela@gmail.com>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/mrxvt/template]: GPL-2.0-or-later

File: srcpkgs/mtpfs/template [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'mtpfs'
 pkgname=mtpfs
 version=1.1
-revision=3
+revision=4
 build_style=gnu-configure
 hostmakedepends="pkg-config"
 makedepends="fuse-devel libmtp-devel libmad-devel libglib-devel libid3tag-devel"
```
Comment (maxice8) [inline srcpkgs/mtpfs/template]: GPL-2.0-or-later

File: srcpkgs/multitail/template [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
+revision=2
 build_style=gnu-makefile
-CFLAGS='-funsigned-char -DLinux -DVERSION=\"\$(VERSION)\" -DCONFIG_FILE=\"\$(CONFIG_FILE)\" -DUTF8_SUPPORT'
-LDFLAGS='-lpanelw -lncursesw -lutil -lm'
 makedepends="ncurses-devel"
+checkdepends="clang-analyzer cppcheck perl"
 short_desc="Tail multiple logfiles"
-maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/multitail/template]: GPL-2.0-only

File: srcpkgs/nausea/template [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'nausea'
 pkgname=nausea
 version=0.3
-revision=1
+revision=2
 build_style=gnu-makefile
 makedepends="ncurses-devel fftw-devel"
 short_desc="A curses audio visualizer"
```
Comment (maxice8) [inline srcpkgs/nausea/template]: BSD-2-Clause

File: srcpkgs/ngetty/template [hunk 6]
```diff
@@ -1,9 +1,9 @@
 # Template file for 'ngetty'
 pkgname=ngetty
 version=1.1
-revision=5
+revision=6
 short_desc="Daemon for virtual console terminals"
-maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/ngetty/template]: GPL-2.0-or-later

File: srcpkgs/netcat/template [hunk 7]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'netcat'
 pkgname=netcat
 version=0.7.1
-revision=6
+revision=7
 build_style=gnu-configure
 configure_args="--program-prefix=g"
 short_desc="The GNU netcat utility"
```
Comment (maxice8) [inline srcpkgs/netcat/template]: GPL-2.0-or-later

File: srcpkgs/obconf/template [hunk 8]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'obconf'
 pkgname=obconf
 version=2.0.4
-revision=5
+revision=6
 build_style=gnu-configure
 hostmakedepends="pkg-config"
 makedepends="gtk+-devel libglade-devel libSM-devel imlib2-devel openbox-devel startup-notification-devel"
```
Comment (maxice8) [inline srcpkgs/obconf/template]: GPL-2.0-or-later

File: srcpkgs/obexfs/template [hunk 9]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'obexfs'
 pkgname=obexfs
 version=0.12
-revision=2
+revision=3
 build_style=gnu-configure
 hostmakedepends="pkg-config"
 makedepends="fuse-devel libbluetooth-devel libobexftp-devel"
```
Comment (maxice8) [inline srcpkgs/obexfs/template]: GPL-2.0-or-later

File: srcpkgs/oroborus/template [hunk 10]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'oroborus'
 pkgname=oroborus
 version=2.0.20
-revision=1
+revision=2
 build_style=gnu-configure
 makedepends="libX11-devel libICE-devel libSM-devel libXext-devel libXpm-devel"
 short_desc="A minimalistic window manager"
```
Comment (maxice8) [inline srcpkgs/oroborus/template]: GPL-2.0-or-later

File: srcpkgs/ogmtools/template [hunk 11]
```diff
@@ -1,19 +1,19 @@
...
 configure_args="--disable-oggtest --disable-vorbistest --with-dvdread
-	--with-dvdread-includes=${XBPS_CROSS_BASE}/usr
-	--with-dvdread-libs=${XBPS_CROSS_BASE}/usr/lib"
+ --with-dvdread-includes=${XBPS_CROSS_BASE}/usr
+ --with-dvdread-libs=${XBPS_CROSS_BASE}/usr/lib"
 hostmakedepends="automake libtool pkg-config"
 makedepends="libvorbis-devel libdvdread-devel"
+short_desc="Tools for information,extractions or creation of OGG media streams"
+maintainer="Martin Riese <grauehaare@gmx.de>"
+license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/ogmtools/template]: GPL-2.0-only

File: srcpkgs/patchutils/template [hunk 12]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'patchutils'
 pkgname=patchutils
 version=0.3.4
-revision=1
+revision=2
 build_style=gnu-configure
 hostmakedepends="perl"
 depends="perl"
```
Comment (maxice8) [inline srcpkgs/patchutils/template]: GPL-2.0-or-later

File: srcpkgs/pipenightdreams/template [hunk 13]
```diff
@@ -1,15 +1,14 @@
...
 version=0.10.0
-revision=1
+revision=2
 build_style=gnu-configure
-configure_args=""
 hostmakedepends="automake flex libtool pkg-config"
 makedepends="SDL_image-devel"
 short_desc="Just another pipe trip"
 maintainer="Jürgen Buchmüller <pullmoll@t-online.de>"
 license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/pipenightdreams/template]: GPL-2.0-or-later

File: srcpkgs/pwnat/template [hunk 14]
```diff
@@ -1,17 +1,17 @@
...
 # the git-hash of v0.3 as it isn't tagged
 _githash=572fcfb76a1b4b46faaa6b36817a39671b6f3c7e
-homepage="http://samy.pl/pwnat/"
-distfiles="https://github.com/samyk/pwnat/archive/${_githash}.tar.gz"
-short_desc="serverless NAT to NAT hole punching"
+wrksrc="${pkgname}-${_githash}"
+build_style=gnu-makefile
+short_desc="Serverless NAT to NAT hole punching"
 maintainer="Enno Boland <gottox@voidlinux.eu>"
 license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/pwnat/template]: GPL-3.0-or-later

File: srcpkgs/pygtksourceview/template [hunk 15]
```diff
@@ -1,19 +1,19 @@
...
+revision=5
 build_style=gnu-configure
 configure_args="--disable-static --disable-docs"
 hostmakedepends="pkg-config intltool python-devel pygtk-devel"
 makedepends="python-devel gtksourceview2-devel pygtk-devel"
 depends="pygtk"
 short_desc="Python bindings for gtksourceview2"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://www.gnome.org"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/pygtksourceview/template]: GPL-2.0-or-later

File: srcpkgs/quvi/template [hunk 16]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'quvi'
 pkgname=quvi
 version=0.4.2
-revision=3
+revision=4
 build_style=gnu-configure
 hostmakedepends="pkg-config"
 makedepends="libquvi-devel"
```
Comment (maxice8) [inline srcpkgs/quvi/template]: LGPL-2.1-or-later

File: srcpkgs/rakarrack/template [hunk 17]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'rakarrack'
 pkgname=rakarrack
 version=0.6.1
-revision=2
+revision=3
 build_style=gnu-configure
 hostmakedepends="alsa-utils"
 makedepends="jack-devel libsndfile-devel libXpm-devel fltk-devel libsamplerate-devel"
```
Comment (maxice8) [inline srcpkgs/rakarrack/template]: GPL-2.0-or-later

File: srcpkgs/rsstail/template [hunk 18]
```diff
@@ -1,9 +1,10 @@
...
 pkgname=rsstail
 version=2.1
-revision=1
+revision=2
 build_style=gnu-makefile
 makedepends="libmrss-devel"
+checkdepends="cppcheck"
 short_desc="More or less an rss reader"
 maintainer="Enno Boland <gottox@voidlinux.eu>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/rsstail/template]: GPL-2.0-only

File: srcpkgs/recordmydesktop/template [hunk 19]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'recordmydesktop'
 pkgname=recordmydesktop
 version=0.3.8.1
-revision=3
+revision=4
 build_style=gnu-configure
 hostmakedepends="pkg-config"
 makedepends="zlib-devel alsa-lib-devel libvorbis-devel libtheora-devel libXext-devel libXdamage-devel libSM-devel"
```
Comment (maxice8) [inline srcpkgs/recordmydesktop/template]: GPL-2.0-or-later

File: srcpkgs/scrotty/template [hunk 20]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'scrotty'
 pkgname=scrotty
 version=2.0
-revision=1
+revision=2
 build_style=gnu-configure
 hostmakedepends="pkg-config auto-auto-complete"
 makedepends="libpng-devel"
```
Comment (maxice8) [inline srcpkgs/scrotty/template]: GPL-3.0-or-later

File: srcpkgs/since/template [hunk 21]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'since'
 pkgname=since
 version=1.1
-revision=1
+revision=2
 short_desc="Stateful tail"
 maintainer="Orphaned <orphan@voidlinux.eu>"
 license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/since/template]: GPL-3.0-or-later

File: srcpkgs/silc/template [hunk 22]
```diff
@@ -11,6 +12,5 @@ short_desc="Secure Internet Live Conferencing server"
 maintainer='Juan RP <xtraeme@voidlinux.eu>'
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/silc/template]: GPL-2.0-or-later

File: srcpkgs/slim/template [hunk 23]
```diff
@@ -1,21 +1,21 @@
...
 configure_args="-DUSE_CONSOLEKIT=1 -DUSE_PAM=yes
  -DFREETYPE_INCLUDE_DIR_freetype2=${XBPS_CROSS_BASE}/usr/include/freetype2"
+conf_files="/etc/slim.conf /etc/pam.d/slim"
 hostmakedepends="pkg-config"
 makedepends="libpng-devel freetype-devel libjpeg-turbo-devel libXrandr-devel
  libXmu-devel libXft-devel pam-devel ConsoleKit2-devel"
-conf_files="/etc/slim.conf /etc/pam.d/slim"
 short_desc="Desktop-independent graphical login manager for X11"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/slim/template]: GPL-2.0-or-later

File: srcpkgs/starplot/template [hunk 24]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'starplot'
 pkgname=starplot
 version=0.95.5
-revision=1
+revision=2
 build_style=gnu-configure
 configure_args="--disable-rpath"
 hostmakedepends="pkg-config"
```
Comment (maxice8) [inline srcpkgs/starplot/template]: GPL-2.0-or-later

File: srcpkgs/tree/template [hunk 25]
```diff
@@ -1,13 +1,13 @@
...
 pkgname=tree
 version=1.7.0
-revision=2
+revision=3
 build_style=gnu-makefile
-homepage="http://mama.indstate.edu/users/ice/tree"
-distfiles="$homepage/src/tree-$version.tgz"
 short_desc="Resursive directory listing program"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/tree/template]: GPL-2.0-or-later

File: srcpkgs/trollock/template [hunk 26]
```diff
@@ -1,15 +1,15 @@
...
-revision=1
+revision=2
 build_style=gnu-configure
-short_desc="Xtrlock fork with PAM support"
-maintainer="Andrea Brancaleoni <abc@pompel.me>"
 hostmakedepends="python pkg-config"
 makedepends="libX11-devel pam-devel"
+short_desc="Xtrlock fork with PAM support"
+maintainer="Andrea Brancaleoni <abc@pompel.me>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/trollock/template]: GPL-2.0-or-later

File: srcpkgs/tsocks/template [hunk 27]
```diff
@@ -1,17 +1,17 @@
...
 version=1.8beta5
-revision=3
+revision=4
 wrksrc="tsocks-1.8"
 build_style=gnu-configure
 configure_args="--libdir=/usr/lib"
+hostmakedepends="automake"
 short_desc="Transparent SOCKS proxying library"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/tsocks/template]: GPL-2.0-only

File: srcpkgs/udevil/template [hunk 28]
```diff
@@ -1,12 +1,12 @@
...
 build_style=gnu-configure
-hostmakedepends="pkg-config intltool"
 configure_args="--disable-systemd"
-makedepends="libglib-devel eudev-libudev-devel"
 conf_files="/etc/udevil/udevil.conf"
+hostmakedepends="pkg-config intltool"
+makedepends="libglib-devel eudev-libudev-devel"
 short_desc="CLI which mounts and unmounts removable devices without a password"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/udevil/template]: GPL-3.0-or-later

File: srcpkgs/unifdef/template [hunk 29]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'unifdef'
 pkgname=unifdef
 version=2.11
-revision=1
+revision=2
 build_style=gnu-makefile
 make_install_args="prefix=/usr"
 short_desc="Selectively remove C preprocessor conditionals"
```
Comment (maxice8) [inline srcpkgs/unifdef/template]: BSD-2-Clause

File: srcpkgs/usbmuxd/template [hunk 30]
```diff
@@ -1,20 +1,20 @@
...
 configure_args="--without-systemd"
-hostmakedepends="pkg-config"
-system_accounts="usbmux"
-usbmux_homedir="/var/lib/lockdown"
 make_dirs="/var/lib/lockdown 0755 usbmux usbmux"
+hostmakedepends="pkg-config"
 makedepends="libusb-devel libimobiledevice-devel"
 short_desc="USB Multiplex Daemon"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-2,GPL-3"
```
Comment (maxice8) [inline srcpkgs/usbmuxd/template]: GPL-2.0-or-later, LGPL-2.1-or-later

File: srcpkgs/udisks/template [hunk 31]
```diff
@@ -1,19 +1,19 @@
...
 hostmakedepends="pkg-config intltool libxslt docbook-xsl glib-devel dbus-glib-devel"
 makedepends="libglib-devel device-mapper-devel
  liblvm2app-devel libparted-devel libatasmart-devel polkit-devel
  dbus-glib-devel libgudev-devel sg3_utils-devel"
 depends="dbus"
-make_dirs="/var/lib/udisks 0750 root root"
 short_desc="Disk Management Service"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://www.freedesktop.org/wiki/Software/udisks"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/udisks/template]: GPL-2.0-or-later

File: srcpkgs/vpnc/template [hunk 32]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'vpnc'
 pkgname=vpnc
 version=0.5.3
-revision=5
+revision=6
 hostmakedepends="perl"
 if [ "$CROSS_BUILD" ]; then
 	hostmakedepends+=' vpnc'
```
Comment (maxice8) [inline srcpkgs/vpnc/template]: GPL-2.0-or-later

File: srcpkgs/vpnc/template [hunk 33]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'vpnc'
 pkgname=vpnc
 version=0.5.3
-revision=5
+revision=6
 hostmakedepends="perl"
 if [ "$CROSS_BUILD" ]; then
```
Comment (maxice8) [inline srcpkgs/vpnc/template]: also please move this  to between the  conf_files= and do_configure()

File: srcpkgs/vorbisgain/template [hunk 34]
```diff
@@ -10,5 +10,5 @@ short_desc="A utility that computes the ReplayGain values for Ogg Vorbis files"
 maintainer="Farhad Shahbazi <grauwolf@geekosphere.org>"
 license="LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/vorbisgain/template]: LGPL-2.1-only

**Comment by maxice8** [comment]
done

**Comment by Chocimier** [comment]
changed

**Comment by maxice8** [comment]
merged

**Pull request closed without merging**